templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -229,35 +229,23 @@ contents:
             exit 1
           fi
           new_conn_file="${new_conn_files[0]}"
-          # modify file to work with OVS and have unique settings
-          sed -i '/^multi-connect=.*$/d' ${new_conn_file}
```

Comment:
I suppose that this one here is not needed?

Reply:
oops no, I will add that back.

I don't see though how it could make any practical difference or how could this profile can be activated multiple times.